pennylane/tape/qubit_param_shift.py

```diff
@@ -379,3 +379,22 @@ def processing_fn(results):
             return np.apply_along_axis(dot, 0, results)
 
         return tapes, processing_fn
+
+    @property
+    def specs(self):

```

Comment:
It's weird but the docstring seems to be empty for `specs` in `QubitParamShiftTape`:
![image](https://user-images.githubusercontent.com/49409390/121568311-93db7f80-c9ed-11eb-91c5-f21380ab9d44.png)

Reply:
copied over from the docstring in `QuantumTape` now.  I thought it would inherit the docstring too....